.env.template

```diff
@@ -46,3 +46,5 @@ FAF_VOTING_USER=root
 FAF_LEGACY_UPDATER_USER=root
 FAF_MONGODB_USER=root
 FAF_NODEBB_USER=root
+FAF_LEAGUE_SERVICE_USER=root
+FAF_LEAGUE_DB_USER=root
```

Comment:
Since neither is supposed to touch the core database, I'm not sure if these are necessary.

Reply:
It depends on whether the service needs to run as root user inside the docker container.